stderr of test 'bad-001` in directory 'tests/i18n` itself:


# 22:24:22 >  
# 22:24:22 >  Mtimeout -timeout 180 Mserver "--config=/opt/MonetDB/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/home/teubner/pf/cvs-head/runtime/.libs:/home/teubner/pf/cvs-head/runtime:/opt/MonetDB/lib/MonetDB" --set "gdk_dbfarm=/home/teubner/pf/cvs-head/dbfarm" --set "sql_logdir=/home/teubner/pf/cvs-head/log" --set mapi_port=30263 --set monet_prompt= --trace "--dbname=mTests_tests_i18n" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 22:24:22 >  


# 22:24:23 >  
# 22:24:23 >  Mtimeout -timeout 60 MapiClient -lxquery -oxml -umonetdb -Pmonetdb --host=localhost --port=30263 bad-001.xq
# 22:24:23 >  

syntax error, unexpected invalid_character on line 5, column 19 (next token is `Ã')
parse error: XQuery parsing failed
# halted in parser.y (PFparse), line 2868

# 22:24:23 >  
# 22:24:23 >  Done.
# 22:24:23 >  

